lib/mods/replace-methods.js

```diff
@@ -30,17 +38,25 @@ function replaceGlobalMethodCalls(node, imports) {
   if (nodeTypes.isPropertyAccessExpression(node) &&
       nodeTypes.isCallExpression(node.getParent()) &&
       isGlobal(node.getExpression())) {
-    const callerType = node.getExpression().getType().getSymbol().getName();
+    let callerType = node.getExpression().getType().getSymbol().getName();
     const method = node.getSymbol().getEscapedName();
-    const original = getOriginal(method, 'method', callerType);
+    let type = 'method';
+
+    if (node.getExpression().getType().getConstructSignatures()[0]) {
```

Comment:
Let's try to save variables in expressions a bit more. I think at least we should have `const baseTypeConstructSignature = node.getExpression().getType().getConstructSignatures()[0]`, but maybe we should also have `base` and `baseType`.

Reply:
I added a baseType, let me know if you think we need a base